whois-client/src/main/java/net/ripe/db/whois/api/syncupdate/SyncUpdateBuilder.java

```diff
@@ -83,23 +84,20 @@ public SyncUpdateBuilder setRedirect(final boolean redirect) {
         return this;
     }
 
-    public Client build() {
-        if (url != null) {
-            return new Client(url, headers, data, help, diff, aNew, redirect);
-        }
-
-        if (host != null && port != null && source != null) {
-            return new Client(host, port, source, headers, data, help, diff, aNew, redirect);
-        }
+    public SyncUpdateBuilder setCharset(final String charset) {
+        this.charset = charset;
+        return this;
+    }
 
-        throw new IllegalStateException("Either (host, port, source) or (url) should not be null");
+    public Client build() {
+        return new Client(url, host, port, source, headers, data, help, diff, aNew, redirect, charset);
     }
 
     public static class Client {
 
         private static final Joiner.MapJoiner PARAM_JOINER = Joiner.on('&').withKeyValueSeparator("=");
         private static final Pattern CHARSET_PATTERN = Pattern.compile(".*;charset=(.*)");
-        private static final String CHARSET = "ISO-8859-1";
+        private static final String DEFAULT_CHARSET = "ISO-8859-1";
```

Comment:
Minor: Maybe consider using `StandardCharsets.ISO_8859_1` instead of hardcoded string

Reply:
Done!